### PR TITLE
feat: Dashboard E2E tests fixed, migrated to playwright

### PR DESCRIPTION
**Pull request by tkonieczny**
Migrating to playwright took less time than handling workarounds for Cypress Comands in case of custom setup.

### CODE REVIEW

File: test/dashboard-e2e/pages/TestExecutionsPage.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    
+    async runTest() {
+        await this.page.click('div[class="ant-page-header-heading"] button')
+    }
+    
+    async openExecutionDetails(executionName) {
+        await this.page.click(`xpath=//tr[.//span[text()="${executionName}"]]`)
+    }
+
+    // validateLogOutputContents(expectedText, customTimeout=null) { //TODO: update
```
Comment (vsukhin) [inline test/dashboard-e2e/pages/TestExecutionsPage.ts:17]: I would remove commented code

File: test/dashboard-e2e/playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    /* Test against branded browsers. */
+    // {
+    //   name: 'Microsoft Edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ..devices['Desktop Chrome'], channel: 'chrome' },
+    // },
```
Comment (vsukhin) [inline test/dashboard-e2e/playwright.config.ts:73]: I would remove commented code

**Review by vsukhin — APPROVED** [review]
minor notes

File: test/dashboard-e2e/playwright.config.ts (hunk 2 above)
Comment (tkonieczny) [reply test/dashboard-e2e/playwright.config.ts:73]: removed

File: test/dashboard-e2e/pages/TestExecutionsPage.ts (hunk 1 above)
Comment (tkonieczny) [reply test/dashboard-e2e/pages/TestExecutionsPage.ts:17]: removed

**Pull request merged**